### PR TITLE
[Content] Replace 577 Ember Raider With A  Korath 582

### PR DESCRIPTION
**Pull request by rovermicrover**
## Summary
The Lor'kas Ik 577 should not be seen outside of Korath home space, so we need to remove it from the Large Ember Raid fleet variants and replace it with a 582 variant. To still allow the 577 to be capturable add the 577 (Jump) to the Korath Home Fleet as a rare variant.

## Save File
[Save File Found Here](https://www.dropbox.com/s/iz2clkddmk6yhox/Rover%20Freewing~~previous-1.txt?dl=0)

### CODE REVIEW

**Comment by Saugia** [comment]
There is a reason that some production ships do have JDs while others don't, and if this were to change, it would be in future Korath content. Now, if enough players believe this to be the case, then perhaps we can change this, but I'll wait on raven's response for that matter.

Just as a heads up before you continue to open PRs on Korath content - everything that was recently released in 9.15 has been put into the game as intended, and those of us involved in the Korath side of content have thought of most content ideas such as this before releasing it. I suggest opening issues regarding any further inquiries you may have, rather than opening and closing PRs.

**Comment by rovermicrover** [comment]
Awesome thanks for the info. 

Is there an official list of who is in charge of what plots and where they hang out?

**Comment by Saugia** [comment]
Ravenshining is generally the lead when it comes to Korath content, and Zitchas has a bit of a say in it too as it has some tie-in with the Remnant. Otherwise, there are a few members in the community on the server, such as me and EjoThims who have done a lot of work with Raven and crew.

File: data/korath/korath variants.txt [hunk 1]
```diff
@@ -923,6 +943,39 @@ ship "Lor'nag Ik 590" "Lor'nag Ik 590 (Troop Transport)"
...
+	gun 45 -253 "Fire-Lance"
+	turret -77 -116.5 "Shunt-Strike Turret"
+	turret 77 -116.5 "Shunt-Strike Turret"
+	turret -87 -83 "Warder Anti-Missile"
+	turret 87 -83 "Banisher Grav-Turret"
+	turret -51 276.5 "Banisher Grav-Turret"
+	turret 51 276.5 "Warder Anti-Missile"
+
+
+
```
Comment (ravenshining) [inline data/korath/korath variants.txt:978]: Nope.

```suggestion
```

**Review by ravenshining — CHANGES_REQUESTED** [review]
While looking this over, I realised I made a slight mistake - it's the KLI 582 (tanker) that should be in the ember waste raids, *not* the LKI 577 (freighter).  So, actually, if you were to remove the LKI 577 from those raids in favour of adding the KLI 582, we can use this PR do do so.  Since you haven't edited those lines, you'll have to do that part.

The LNI 590, on the other hand, should *not* be capturable outside of missions.  It's too powerful a boarding vessel to be handed out at the start of the game; you can work for a couple as spoils of mission completion - albiet with consequences to various campaigns - but I don't want to do more than that if we can help it.  Also, the (Ember) variant you've added is redundant with the (Troop Transport) variant.

I'm on the fence as to whether the LKI 577 should be capturable in the wild.  I'm worried about its plundering potential given its large hold... and there isn't any real reason for it to leave Korath space, except in Wanderers and some missions Zitchas and I have planned but haven't written yet.  However, some have argued that it could be used to make jumps between isolated sections of Korath space.  Lore-wise I think they shouldn't, but if it makes sense gameplay-wise for them to be capturable, giving JDs to the LKI 577s in "Korath Home" should do the trick.

**Comment by rovermicrover** [comment]
@ravenshining Updated off your feedback

* 590 Removed
* 577 Ember removed and 577 Jump added to Korath Home as a very rare fleet variant.

**Comment by rovermicrover** [comment]
Out and about and working off my steam deck so will load and re verify this in game tonight just in case.

**Comment by rovermicrover** [comment]
Verified locally this should be GTG

File: data/korath/korath variants.txt [hunk 2]
```diff
@@ -859,6 +825,26 @@ ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Jump)"
...
+		"Liquid Sodium Cooler"
+		"Fuel Processor" 5
+		"Microbot Defense Station" 5
+		"Thermal Repeater Rifle" 551
+		
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Jump Drive"
+		Hyperdrive
+
```
Comment (ravenshining) [inline data/korath/korath variants.txt:845]: Make sure the turrets get properly distributed to sensible hardpoints.

```suggestion

	turret "Banisher Grav-Turret"
	turret "Warder Anti-Missile"
	turret "Warder Anti-Missile"
	turret "Banisher Grav-Turret"
	turret "Langrage Hyper-Heaver"
	turret "Langrage Hyper-Heaver"
	turret "Banisher Grav-Turret"
	turret "Warder Anti-Missile"

```

File: data/korath/korath.txt [hunk 3]
```diff
@@ -279,15 +279,18 @@ fleet "Korath Home"
 	cargo 3
 	personality
 		timid opportunistic
-	variant 3
+	variant 90
```
Comment (ravenshining) [inline data/korath/korath.txt:282]: I'm no numbers person but those are some extreme odds right there.

```suggestion
	variant 30
```

File: data/korath/korath.txt [hunk 4]
```diff
@@ -279,15 +279,18 @@ fleet "Korath Home"
...
 		"Lor'kas Ik 577"
 		"'olofez" 3
-	variant 1
+	variant 30
```
Comment (ravenshining) [inline data/korath/korath.txt:285]:
```suggestion
	variant 10
```

File: data/korath/korath.txt [hunk 5]
```diff
@@ -279,15 +279,18 @@ fleet "Korath Home"
...
 		"Kas'lor Ik 582"
 		"'olofez" 3
-	variant 1
+	variant 30
```
Comment (ravenshining) [inline data/korath/korath.txt:288]:
```suggestion
	variant 10
```

**Review by ravenshining — CHANGES_REQUESTED** [review]

File: data/korath/korath.txt (hunk 3 above)
Comment (rovermicrover) [reply data/korath/korath.txt:282]: The frequency for this fleet to spawn a lot more frequent. The numbers present make sure that the 577 in its new fleet doesn’t spawn more often. Keep its original rareness.

**Review by ravenshining — APPROVED** [review]

**Review by Zitchas — APPROVED** [review]

**Comment by warp-core** [comment]
Can the title and summary be updated? It seems like the goal of this PR has changed quite a bit from what it was originally going for but it doesn't look like the title or summary were updated accordingly.

**Pull request merged**